spinta/backends/__init__.py

```diff
@@ -34,7 +34,9 @@ def prepare(context: Context, type: Object, backend: Backend, value: dict) -> di
     # prepare objects and it's properties for datastore
     new_loaded_obj = {}
     for k, v in type.properties.items():
-        new_loaded_obj[k] = prepare(context, v.type, backend, value[k])
+        # only load value keys which are available in schema
+        if k in value:
+            new_loaded_obj[k] = prepare(context, v.type, backend, value[k])
```

Comment:
If `k` is not in `value`, then value should be `NA`. Now if you skip a property entirely, `v.type` does have a chance to prepare NA values. You can add a fixme comment and some one will fix it later.

Reply:
Also, I think, `prepare` will always have all properties, so probably there is no need to check if `k` is in `value` at all.